frontend/app/controllers/spree/checkout_controller.rb

```diff
@@ -150,7 +150,8 @@ def ensure_order_not_completed
 
     def ensure_sufficient_stock_lines
       if @order.insufficient_stock_lines.present?
-        flash[:error] = Spree.t(:inventory_error_flash_for_insufficient_quantity)
+        out_of_stock_items = @order.insufficient_stock_lines.map { |l| "\"#{l.name}\"" }.join(', ')
```

Comment:
No need to convert name into a string, it is already one. 

Also I would prefer to join the names with Rails' `to_sentence` instead of `join` with comma.